src/fetch-map/layer-map.ts

```diff
@@ -495,11 +496,11 @@ export function calculateClusterRadius(
 export function getDefaultAggregationExpColumnAliasForLayerType(
   layerType: LayerType,
   provider: ProviderType,
-  columns: string[]
+  schema: SchemaField[]
 ): string {
-  if (columns && layerType === 'clusterTile') {
+  if (schema && layerType === 'clusterTile') {
```

Comment:
Ouch, I didn't remeber that, apologies for not highlighting it in the previous PR 🙏

Reply:
I probably should have called it out as a copy/paste except for this change, sorry!